p2p/pkg/depositmanager/store/store_test.go

```diff
@@ -91,8 +91,8 @@ func TestStore_RefundBalanceIfExists(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	refundAmount := big.NewInt(5)
-	err = s.RefundBalanceIfExists(bidder, provider, refundAmount)
+	increaseAmount := big.NewInt(5)
+	err = s.RefundBalanceIfExists(bidder, provider, increaseAmount)
```

Comment:
The changes look good now. But this DB entry can be modified to not have provider address. It will have the same provider address for all entries. So we can save on the extra storage.

Reply:
https://github.com/primev/mev-commit/commit/72e608c5a63dd0d7f1b57c6a84c4bbf227ddf6fc